requirements.txt

```diff
@@ -1,8 +1,8 @@
 # TODO: put back the standard pypi release when
 # next point version of openwisp-controller is released
 openwisp-controller @ https://github.com/openwisp/openwisp-controller/tarball/master
-# openwisp-utils[rest] is left unpinned so the version used
-# is the one dictated by openwisp-controller
+# TODO: Remove before release
```

Comment:
Need to change before merge.

Reply:
I think this was merged and you should be able to change this, right?